lib/nerves_wpa_supplicant/messages.ex

```diff
@@ -17,6 +17,9 @@ defmodule Nerves.WpaSupplicant.Messages do
   def encode(cmd) when is_atom(cmd) do
     to_string(cmd)
   end
+  def encode({:SET_NETWORK, net_id, :key_mgmt, string}) when is_binary(string) do
+    "SET_NETWORK #{net_id} key_mgmt #{string}"
```

Comment:
I think we could maintain consistency with the rest of the code if we either wrap the string in quotes like this:

```elixir
def encode({:SET_NETWORK, net_id, :key_mgmt, string}) when is_binary(string) do
  "SET_NETWORK #{net_id} key_mgmt \"#{string}\""
end
```

or, perhaps even better, just convert the one thing and depend on the rest of the code to do what it does consistently:

```elixir
def encode({:SET_NETWORK, net_id, :key_mgmt, key_mgmt}) when is_string(key_mgmt) do
  encode({:SET_NETWORK, net_id, :key_mgmt, String.to_atom(key_mgmt)})
end
```